src/status_im/utils/utils.cljs

```diff
@@ -84,14 +88,21 @@
       (str/trim)))
 
 (defn first-index
-  [cond coll]
+  "Returns first index in coll where cond? on coll element is truthy"
+  [cond? coll]
   (loop [index 0
-         cond cond
          coll coll]
     (when (seq coll)
-      (if (cond (first coll))
+      (if (cond? (first coll))
         index
-        (recur (inc index) cond (next coll))))))
+        (recur (inc index) (next coll))))))
 
 (defn hash-tag? [s]
   (= \# (first s)))
+
+(defn update-if-present
```

Comment:
I'm wondering if there is no standard pattern in the clojure community.
If not we might want some basic tests for this.